ros_ign_bridge/test/subscribers/ros_subscriber.cpp

```diff
@@ -239,6 +239,18 @@ TEST(ROSSubscriberTest, TransformStamped)
   EXPECT_TRUE(client.callbackExecuted);
 }
 
+/////////////////////////////////////////////////
+TEST(ROSSubscriberTest, TF2Message)
```

Comment:
Did you mean to also add the "double" tests? You've added support for double in a few places.

Reply:
yes but I forgot to add the test and most of the implementation as well. My bad sorry. It should be fixed now